src/Symfony/Component/HttpClient/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ CHANGELOG
  * added `StreamableInterface` to ease turning responses into PHP streams
  * added `MockResponse::getRequestMethod()` and `getRequestUrl()` to allow inspecting which request has been sent
  * added `EventSourceHttpClient` a Server-Sent events stream implementing the [EventSource specification](https://www.w3.org/TR/eventsource/#eventsource)
+ * allow overriding curl options in `CurlHttpClient`
```

Comment:
* added option "extra.curl" to allow setting additional curl options in `CurlHttpClient`

Reply:
fixed